llama-index-integrations/vector_stores/llama-index-vector-stores-wordlift/README.md

```diff
@@ -1 +1,24 @@
-# LlamaIndex Vector_Stores Integration: Wordlift
+<picture>
+  <img alt="#WordLift" width="200" src="https://eacn2n47zot.exactdn.com/wp-content/uploads/2022/12/logo.svg" style="margin-bottom: 25px;">
```

Comment:
@ossaleon what is this URL https://eacn2n47zot.exactdn.com/wp-content/uploads/2022/12/logo.svg who's managing it?

Please use a domain managed by ourselves, e.g. https://wordlift.io/wp-content/uploads/2022/12/logo.svg ?